docs/docs/releasing.md

```diff
@@ -24,4 +24,9 @@ title: Releasing
 
 ## Publishing
 
-1. GitHub Action will create a new release, review and edit it at https://github.com/eraser-dev/eraser/releases
\ No newline at end of file
+1. GitHub Action will create a new release, review and edit it at https://github.com/eraser-dev/eraser/releases
+
+## Notifying
+
+1. Send an email to the [Eraser mailing list](https://groups.google.com/g/eraser-dev) announcing the release, with links to GitHub.
```

Comment:
this is a good idea, it would be cool to automate these. obviously, out of scope for this PR 😀

Reply:
it can be a reusable generic automation